web_responsive/models/res_users.py

```diff
@@ -24,3 +24,20 @@ class ResUsers(models.Model):
         default="milk",
         required=True,
     )
+    is_redirect_home = fields.Boolean(
+        string="Redirect to Home",
+        help="Redirect to dashboard after signing in",
+        compute="_compute_redirect_home",
+        store=True,
+        readonly=False,
+    )
+
+    @api.depends("action_id")
+    def _compute_redirect_home(self):
+        """
+        Set is_redirect_home to False
+        when action_id has a value.
+        :return:
+        """
+        for user in self:
+            user.is_redirect_home = False if user.action_id else user.is_redirect_home
```

Comment:
Little code improvement:

```suggestion
        self.filtered("action_id").is_redirect_home = False
```

Reply:
@pedrobaeza 
Updated as per the suggestion.